QandA/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Meetings(models.Model):
+    meeting_id = models.AutoField(primary_key=True)
```

Comment:
This declaration should come after declaring all constant static variables for class like `VENUE_CHOICES`
Why?
- Similar block of elements, e.g. declaring fields should be grouped.
- It is best practice to define static variables at top of class.

Reply:
Done, will keep this in mind.